### ISSUE TITLE
Update the docs with a link to this pair programming video

### ISSUE THREAD

**Issue by csells** [csells/go_router#127]
video here: https://twitter.com/mhadaily/status/1455580280924876802

MD link format for "hosted" YT video:

```text
[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg)](https://www.youtube.com/watch?v=YOUTUBE_VIDEO_ID_HERE)
```

**Comment by csells** [comment]
[![Flutter Navigator 2.0 made easy with go_router](https://img.youtube.com/vi/G41NdhieUoc/0.jpg)](https://www.youtube.com/watch?v=G41NdhieUoc)

**Comment by csells** [comment]
fixed in v2.2.6
